pkg/storage/bucket/gcs/config.go

```diff
@@ -25,5 +25,5 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 // RegisterFlagsWithPrefix registers the flags for GCS storage with the provided prefix
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	f.StringVar(&cfg.BucketName, prefix+"gcs.bucket-name", "", "GCS bucket name")
-	f.Var(&cfg.ServiceAccount, prefix+"gcs.service-account", "JSON representing either a Google Developers Console client_credentials.json file or a Google Developers service account key file. If empty, fallback to Google default logic.")
+	f.Var(&cfg.ServiceAccount, prefix+"gcs.service-account", "JSON either from a Google Developers Console client_credentials.json file, or a Google Developers service account key. Needs to be valid JSON, not a filesystem path. If empty, fallback to Google default logic.")
```

Comment:
> If empty, fallback to Google default logic.

I think we need to clarify this. We should check what's the Google SDK default logic and explain it in plain text. I don't know what's that, so our users don't know neither.

Reply:
pushed the change in https://github.com/grafana/mimir/pull/1690/commits/0ffa496476000f55ff60d40096775510c290b197, PTAL